diff --git a/node_modules/@github/combobox-nav/dist/index.js b/node_modules/@github/combobox-nav/dist/index.js
index 5eeaa65..d266601 100644
--- a/node_modules/@github/combobox-nav/dist/index.js
+++ b/node_modules/@github/combobox-nav/dist/index.js
@@ -34,7 +34,7 @@ export default class Combobox {
         this.input.addEventListener('compositionend', this.compositionEventHandler);
         this.input.addEventListener('input', this.inputHandler);
         this.input.addEventListener('keydown', this.keyboardEventHandler);
-        this.list.addEventListener('click', commitWithElement);
+        this.list.addEventListener('mousedown', commitWithElement);
         this.resetSelection();
     }
     stop() {
@@ -44,7 +44,7 @@ export default class Combobox {
         this.input.removeEventListener('compositionend', this.compositionEventHandler);
         this.input.removeEventListener('input', this.inputHandler);
         this.input.removeEventListener('keydown', this.keyboardEventHandler);
-        this.list.removeEventListener('click', commitWithElement);
+        this.list.removeEventListener('mousedown', commitWithElement);
     }
     indicateDefaultOption() {
         var _a;
@@ -162,7 +162,7 @@ function commit(input, list) {
         return false;
     if (target.getAttribute('aria-disabled') === 'true')
         return true;
-    target.click();
+    fireCommitEvent(target)
     return true;
 }
 function fireCommitEvent(target, detail) {
